platform/mellanox/.gitignore

```diff
@@ -16,6 +16,9 @@ sdk-src/*/*
 */*.gz
 */*.egg-info
 
+# Exceptions
```

Comment:
why this is needed if the file is not part of the git?

Reply:
This is needed for BIOS package to be visible for the git.

Our default configuration is to ignore `*.gz` files:
```
*/*.gz
```
This means that the exception needs to be added for the BIOS directory:
```
!bios/*.gz
```

Without having this configuration, we won't be able to add the file to the repository.
This might be useful for the private builds, when all the files are part of the repository.